src/Administration/Resources/app/administration/src/module/sw-settings-rule/page/sw-settings-rule-list/index.js

```diff
@@ -187,38 +181,6 @@ export default {
     },
 
     methods: {
-        setAggregations(criteria) {
```

Comment:
would that not be considered as a break? :thinking:

Reply:
Only if this component would be public, but luckily it's not. Otherwise you are right and it would break anything that relied on this to exists

Reply:
ah nice, haven't looked for a private annotation :ok_hand: